### DISCUSSION TITLE
Not Sure How To Bring Over Existing 2FA entries from Google Authenticator

### DISCUSSION THREAD

**Discussion by jameshfischer** [Authenticator-Extension/Authenticator#811] [Q&A] [answered]
<Do not use this to ask about lost codes or accounts>
<More info here: https://authenticator.cc/docs/en/lost-codes>
<You can also open an issue by Tweeting @AuthExtension>

Perhaps I am just dense, but I was looking for a way to add the existing entries from Google Authenticator (on an Android phone) to this new set up for Edge and Firefox.  When using phones, one creates a QR code to "export accounts", and reads that QR code with the new authenicator app on the new phone, and the two are now "synched", with identical 2FA access abilities to the same logons.

How to accomplish this with this app?

**Comment by mymindstorm** [comment]
Settings -> Backup -> Import -> Import QR Image Backup

**Comment by jameshfischer** [comment]
When taking that approach with an image of the QR code produced with Google Authenticator's "Transfer Accounts - Export Accounts" feature, the error is "Authenticator: 2FA Client Unrecognized QR code".  Does this thing read these QRs?  Or must I transfer from Google to some other Authenticator, and THEN to this app?

**Reply by S4adam** [reply]
How did you end up doing it?

**Comment by jameshfischer** [comment, marked as answer]
It simply does not work.
Let me detail - I used Google Authenticator to generate QR codes, and took a photo of those 2 QR codes with ANOTHER phone, as the Google Authenticator app does not permit one to screenshot the QR codes.  Those well-focused, hi-rez images of the QR codes were fed into the "import" process as described by @mymindstorm above, and the app gave the error "Unrecognized QR code".  So, this a bug - this app is simply unable to read a Google Authenticator Generated QR code, and cannot be used with phones running Google Authenticator.

Bug report filed, as your comment reminded me that it simple does not work as claimed.
